Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -321,9 +321,12 @@ - (void)performSwitchRequest:(NSURL *)appSwitchURL paymentType:(BTPayPalPaymentT
 
     NSURLComponents *urlComponents = [NSURLComponents componentsWithURL:appSwitchURL resolvingAgainstBaseURL:NO];
 
+    // Verify what bt_int_type does
     NSString *queryForAuthSession = [urlComponents.query stringByAppendingString:@"&bt_int_type=2"];
     urlComponents.query = queryForAuthSession;
 
+    // Verify this URL matches v4 URLs
+    // Verify that RDA data is being sent correctly
```

Comment:
We should be good on this front. In both flows - we're sending the RDA data at the same time: after fetching the URL from `paypal_hermes`, before sending analytics data, and before continuing on with the flow via `handlePayPalRequest()`.

In v4, we sent RDA data [here](https://github.com/braintree/braintree_ios/blob/6da9607d732e45df512bc82a5a5c7c2461d9d2c2/BraintreePayPal/PayPalOneTouch/Models/PPOTSwitchRequest.m#L44) and were [calling](https://github.com/braintree/braintree_ios/blob/6da9607d732e45df512bc82a5a5c7c2461d9d2c2/BraintreePayPal/BTPayPalRequestFactory.m#L15) `tokenFromApprovalURL` function to extract the `pairingID`. 

In v5, we both extract the `pairingID` and send the RDA data entirely in `BTPayPalDriver` [here](https://github.com/braintree/braintree_ios/blob/e653dba9e975272112547d3d72a6ea31cc99d72a/Sources/BraintreePayPal/BTPayPalDriver.m#L256-L258).